terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf

```diff
@@ -1,114 +1,112 @@
-resource "kubernetes_config_map" "account_view" {
+resource "kubernetes_config_map" "account_monitoring" {
   metadata {
-    name      = "accountview-dashboard"
+    name      = "accountmonitoring-dashboard"
     namespace = "monitoring"
     labels = {
-      cloud_dashboards = "account_view"
+      cloud_dashboards = "account_monitoring"
     }
   }
 
   data = {
-    "account_view.json" = "${file("../../../../grafana-dashboards/account_view.json")}"
+    "acoount_monitoring.json" = "${file("../../../../grafana-dashboards/account_monitoring.json")}"
```

Comment:
@stafot type here

Reply:
it just changed position @stylianosrigas.  Account view is below.

Reply:
```suggestion
    "account_monitoring.json" = "${file("../../../../grafana-dashboards/account_monitoring.json")}"
```